lib/slurp/application.ex

```diff
@@ -20,5 +20,17 @@ defmodule Slurp.Application do
     Enum.each(new_head_subscriptions, &Slurp.NewHeads.Subscriptions.put/1)
     {:ok, log_subscriptions} = Slurp.Logs.Subscriptions.from_config(blockchains)
     Enum.each(log_subscriptions, &Slurp.Logs.Subscriptions.put/1)
+
+    with b <- blockchains
+      |> Enumerati.filter([start_on_boot: true])
+      |> Enum.map(fn m -> Map.get(m, :id) end)
```

Comment:
This should use struct attribute access. Also how do you feel about using the anonymous function shorthand? e.g.

```elixir
|> Enum.map(& &1.id)
```

Reply:
Im fine with the shorthand here, will swap.